src/test/java/io/appium/java_client/ios/XCUIAutomationTest.java

```diff
@@ -31,8 +34,6 @@
 
 import java.io.File;
 
-import static org.junit.Assert.assertEquals;
-
 public class XCUIAutomationTest {
```

Comment:
I added the test. It doesn't do any assertions because there isn't anything to check, it just calls the methods to verify that there aren't any exceptions.